src/advocate.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Seeking: Developer Advocate"
+---
+
+### Our Mission
+
+Windmill Engineering is building the platform for cloud-native development. Imagine if every time you hit save, you could kick off workflows that run in the cloud, with results that are accessible across your team instantly and historically. How much more productive could you be with faster workflows? What workflows have we not even thought of yet?
+
+### About Us
+
+This is a big mission, and we're a small team (6 people, based in NYC's Union Square).
+
+We're software engineers and toolsmiths. We've built developer tools and consumer products. Nick and Dan (founders) met working at Google on projects like Google Sheets, Google Code, and others. We think cloud-native products like GitHub and Google Sheets are great because of the new workflows they enable. We're excited to be working towards it.
+
+We want to be a better company in an industry that's been neither diverse nor inclusive. Each month, we give $100/employee to an org that's making tech less toxic. We recognize "meritocracy" and "work hard, play hard" as culture smells.
+
+### Our First Product
+
+Tilt relieves the pains of local Kubernetes microservice development. It already automates and optimizes container image builds, and has a Heads-Up Display UI that saves you from having to play 20 questions with kubectl. If you use Kubernetes for production, you need Tilt for development.
+
+We need a Developer Advocate who can help us build a community of Tilt users. We know this can go by many names (DevRel, Dev Evangelism, etc.), but we know we aren't great at it. We love Ashley McNamara's [post describing the role](https://medium.com/@ashleymcnamara/what-is-developer-advocacy-3a92442b627c): "the real goal of a Developer Advocate is to become the voice of the user. We gather feedback in a way developers can’t (since they know the codebase too well), then use that feedback to shape the product to become what it needs to be."
+
+### What You'll Do
```

Comment:
i might also add something about how Tilt is an early product. What dev communities are we missing out on because of our own biases & blind-spots, but would benefit from a cloud-native dev enviornment?